.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 inventory/vagrant_ansible_inventory
 inventory/group_vars/fake_hosts.yml
+inventory/host_vars/k8s-0*/
```

Comment:
Yeah, I know.  I was simply trying to explicitly target the vagrant inventory hosts, in the event we choose to add host inventory down the road.